tests_v2/test_workspace_create_background_starts_terminal.py

```
@@ -64,18 +66,65 @@ def main() -> int:
                 pass
-            if created_workspace:
+            try:
+                layout_marker_path.unlink()
+            except FileNotFoundError:
+                pass
+            for workspace_id in reversed(created_workspaces):
                 try:
-                    c.close_workspace(created_workspace)
+                    c.close_workspace(workspace_id)
                 except Exception:
                     pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)